cmd/buildah/bud.go

```diff
@@ -96,10 +96,20 @@ func budCmd(c *cobra.Command, inputArgs []string, iopts budResults) error {
 			tags = tags[1:]
 		}
 	}
+
+	pullType, err := buildahcli.ValidatePullType(iopts.Pull)
+	if err != nil {
+		return err
+	}
+
 	pullPolicy := imagebuildah.PullNever
-	if iopts.Pull {
+	if pullType == buildahcli.PullImageMissing {
 		pullPolicy = imagebuildah.PullIfMissing
 	}
+	if pullType == buildahcli.PullImageAlways {
```

Comment:
Combine these together with the next section

if pullType == buildahcli.PullImageAlways || iopts.PullAlways {
		pullPolicy = imagebuildah.PullAlways
	}

Reply:
Please hide the pullalways code and remove it from man pages.
Also throw an error if the user says to --pull-always and --pull does not match.